cuqi/testproblem/_testproblem.py

```diff
@@ -419,25 +419,40 @@ class Poisson_1D(BayesianProblem):
     Parameters
     ------------
     dim : int
-        size of the grid for the poisson problem
+        | size of the grid for the poisson problem
 
     endpoint : float
-        Location of end-point of grid.
+        | Location of end-point of grid.
     
     source : lambda function
-        Function for source term.
+        | Function for source term.
 
     field_type : str or cuqi.geometry.Geometry
-        Field type of domain.
+        | Field type of domain. The accepted values are:
+        | a Geometry object.
+        | "KL": a :class:`cuqi.geometry.KLExpansion` geometry object will be created and set as a domain geometry.
+        | "KL_Full": a :class:`cuqi.geometry.KLExpansion_Full` geometry object will be created and set as a domain geometry.
+        | "Step": a :class:`cuqi.geometry.StepExpansion` geometry object will be created and set as a domain geometry.
+        | "CustomKL": a :class:`cuqi.geometry.CustomKL` geometry object will be created and set as a domain geometry.
+        | None: a :class:`cuqi.geometry.Continuous1D` geometry object will be created and set as a domain geometry.
 
-    KL_map : lambda function
-        Mapping used to modify field.
+    field_params : dict
+        | Passed as argument `params` to the underlying geometry when field type is `KL` or `KL_Full` or `CustomKL`.
 
-    KL_imap : lambda function
-        Inverse of KL map.
+    map : lambda function
+        | Mapping used to modify field.
+
+    imap : lambda function
+        | Inverse of KL map.
 
     SNR : int
-        Signal-to-noise ratio
+        | Signal-to-noise ratio
+
+    n_steps: int, default 3
+        | Number of steps for the "Step" field.
+
+    observation_nodes : lambda function
```

Comment:
Would observation_map be more clear?

Reply:
I am not sure what is the best name here. I wanted to emphasize that this map takes the grid as input and spits out (subdomain) of the grid, do you think something like `observation_grid_map` is good, or we can use `observation_map`. My concern about the latter is that it might be confused with a general map (similar to Heat1D parameters: map and imap), what do you think? suggest?

Reply:
I like observation grid map. Makes it a bit more clear

Reply:
Great, I use this name now